server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/SpamAssassinListenerModule.java

```diff
@@ -36,8 +36,8 @@
 import com.google.inject.Provides;
 import com.google.inject.Scopes;
 
-public class SpamAssassinMailboxModule extends AbstractModule {
-    private static final Logger LOGGER = LoggerFactory.getLogger(SpamAssassinMailboxModule.class);
+public class SpamAssassinListenerModule extends AbstractModule {
```

Comment:
and here we should just call MailboxManager.addGlobalListener(SpamEventListener)